libs/client/client.ts

```diff
@@ -0,0 +1,113 @@
+import axios, { AxiosResponse } from "axios";
+import path from "path";
+import { parentLogger } from "../utils";
```

Comment:
Hello Israel, many thanks for your thorough review, it is appreciated.

I'm quite new to Pino and I was trying to figure out how to log "file location". I came across this [thread on Stackoverflow](https://stackoverflow.com/questions/61803101/pino-pretty-how-to-add-file-name-to-log-line) and followed their approach (wich uses `parentLogger`).

I believe you have left a few comments related to this topic, so feels like it deserves an issue of it's own. I will repurpose #33 so that we can discuss logging further.